http-clients/src/main/java/com/palantir/remoting/http/errors/SerializableErrorErrorDecoder.java

```diff
@@ -101,7 +106,15 @@ public Exception decode(String methodKey, Response response) {
         }
     }
 
-    private static Exception constructException(Class<? extends Exception> exceptionClass, String message) {
+    private static Exception constructException(Class<? extends Exception> exceptionClass, String message, int status) {
```

Comment:
looking at http://grepcode.com/file/repo1.maven.org/maven2/javax.ws.rs/javax.ws.rs-api/2.0.1/javax/ws/rs/WebApplicationException.java#65 I think this method ought to simply invoke:

    return constructException(exceptionClass, message, null, status);

...and that that would be a lot less duplicate code.